src/config/testnet/home.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2018-present, Leap DAO (leapdao.org)
+ *
+ * This source code is licensed under the GNU GENERAL PUBLIC LICENSE Version 3
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import AppLayout from '../../components/appLayout';
+
+@observer
+export default class Home extends React.Component {
+  render() {
+    return (
+      <AppLayout section="home">
+        <div style={{ maxWidth: '500px' }}>
```

Comment:
Why not markdown?

Reply:
good question. In the beggining I had a fuzzy idea that home screen may be complex (e.g. have two columns) and it will be easier to code that directly in JSX. However, later on the page pan out a simple text blanket, so, yes, markdown would be better herer